tests/beaker_tests/tacacs_global/tacacs_global_provider_defaults.rb

```diff
@@ -75,7 +72,7 @@
 
     # Expected exit_code is 2 since this is a puppet agent cmd with change.
     cmd_str = PUPPET_BINPATH + 'agent -t'
-    on(agent, cmd_str, acceptable_exit_codes: [2])
+    on(agent, cmd_str, acceptable_exit_codes: [0, 2])
```

Comment:
Will this ever make a change, now?  If not, is it needed anymore?  No matter what, the comment is no longer valid.